src/codeGenerator/triggerEvaluator.ts

```diff
@@ -22,9 +22,8 @@ export default class TriggerEvaluator implements TriggerVisitor {
     visitLoad(astNode: Load) {
         throw new DSLSyntaxError(astNode, "load and tick triggers cannot be combined");
```

Comment:
This should probably just say `load` now.

Reply:
ack